webapp/components/notifications/NotificationMenu/index.vue

```diff
@@ -17,30 +17,9 @@
 </template>
 
 <script>
-import NotificationList from '../NotificationList'
 import Dropdown from '~/components/Dropdown'
-import gql from 'graphql-tag'
-
-const MARK_AS_READ = gql(`
-mutation($id: ID!, $read: Boolean!) {
-  UpdateNotification(id: $id, read: $read) {
-    id
-    read
-  }
-}`)
-
-const NOTIFICATIONS = gql(`{
-  currentUser {
-    id
-    notifications(read: false, orderBy: createdAt_desc) {
-      id read createdAt
-      post {
-        id createdAt disabled deleted title contentExcerpt slug
-        author { id slug name disabled deleted }
-      }
-    }
-  }
-}`)
+import { currentUserNotificationsQuery, updateNotificationMutation } from '~/graphql/User'
+import NotificationList from '../NotificationList/NotificationList'
 
 export default {
   name: 'NotificationMenu',
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-15T23:04:53Z" title="Friday, August 16th 2019, 1:04:53 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -17,30 +17,80 @@
 </template>
 
 <script>
-import NotificationList from '../NotificationList'
+import NotificationList from '../NotificationList/NotificationList'
 import Dropdown from '~/components/Dropdown'
 import gql from 'graphql-tag'
 
-const MARK_AS_READ = gql(`
-mutation($id: ID!, $read: Boolean!) {
-  UpdateNotification(id: $id, read: $read) {
-    id
-    read
+const MARK_AS_READ = gql`
+  mutation($id: ID!, $read: Boolean!) {
+    UpdateNotification(id: $id, read: $read) {
+      id
+      read
+    }
   }
-}`)
+`
 
-const NOTIFICATIONS = gql(`{
-  currentUser {
-    id
-    notifications(read: false, orderBy: createdAt_desc) {
-      id read createdAt
-      post {
-        id createdAt disabled deleted title contentExcerpt slug
-        author { id slug name disabled deleted }
+const NOTIFICATIONS = gql`
```


Move that into `~/graphql`

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-16T07:54:43Z" title="Friday, August 16th 2019, 9:54:43 am +02:00">Aug 16, 2019</time>_

---


Done +++ …